core/sail/fts/elasticsearch/pom.xml

```diff
@@ -15,6 +15,8 @@
 
 	<properties>
 		<elasticsearch.version>1.7.0</elasticsearch.version>
+                <log4j.version>2.6.2</log4j.version>
+                <fasterxml.jackson.version>2.8.3</fasterxml.jackson.version>
```

Comment:
Also: the jackson version is declared at the project root pom level, it should not be set here.